packages/edit-site/src/components/block-editor/index.js

```diff
@@ -133,6 +143,7 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 			...restStoredSettings,
 			__experimentalBlockPatterns: blockPatterns,
 			__experimentalBlockPatternCategories: blockPatternCategories,
+			__experimentalFetchPageEntities: fetchPagesEntities,
```

Comment:
We're using `fetchPagesEntities` but it isn't in the memo dependencies. Is that intended?

Reply:
Good catch!